src/spetlr/transformers/country_to_alphacode_transformer_nc.py

```diff
@@ -0,0 +1,39 @@
+import pycountry
+import pyspark.sql.functions as F
+import pyspark.sql.types as T
+from pyspark.sql import DataFrame
+
+from spetlr.etl import TransformerNC
+
+
+def translate_country_to_alpha2(country_name: str):
+    return pycountry.countries.get(name=country_name).alpha_2
+
+
+translateUDF = F.udf(lambda z: translate_country_to_alpha2(z), T.StringType())
+
+
+class CountryToAlphaCodeTransformerNC(TransformerNC):
+    def __init__(self, *, col_name: str, output_col_name: str = None):
+        """
+        A simple transformer to translate country names to alpha-2 codes
+
+        Args:
+            col_name: The name of the column with country names
+            output_col_name: The name of the column to create,
+            defaults to col_name if none is given
+        """
+        super().__init__()
```

Comment:
In the other transformerNC inheritances, this code is added:

```python 
        dataset_input_keys: Union[str, List[str]] = None,
        dataset_output_key: str = None,
    ):
        super().__init__(
            dataset_input_keys=dataset_input_keys,
            dataset_output_key=dataset_output_key,
        )
```

Should this class have the same?